Blockzilla/BrowserViewController.swift

```diff
@@ -819,7 +820,22 @@ extension BrowserViewController: URLBarDelegate {
     func urlBar(_ urlBar: URLBar, didEnterText text: String) {
         // Hide find in page if the home view is displayed
         let isOnHomeView = homeView != nil
-        overlayView.setSearchQuery(query: text, animated: true, hideFindInPage: isOnHomeView)
+        if Settings.getToggle(.enableSearchSuggestions) {
+            searchSuggestClient.getSuggestions(text,callback: {suggestions, error in
+                if suggestions == nil {
+                    //self.overlayView.setSearchQuery(queryArray: [], animated: true, hideFindInPage: true)
```

Comment:
To show that it won't always run the same code regardless of conditionals. I split this up into 2 PRs because it's 2 different issues. The comments are what will happen when the OverlayView is ready for arrays.

Reply:
Gotcha, thank you for the clarification on that! 😄